README_github.md

```diff
@@ -0,0 +1,37 @@
+Scribus
+=======
+
+**Note:** This exists as a mirror of Scribus for the Scribus developer community to collaborate on Scribus independently from the main team who will continue to develop on svn.  You can find the official page for Scribus at http://www.scribus.net
+
+Why Scribus on GitHub?
+----------------------
+- GitHub has become a way to quickly and ingeniously collaborate.
+- Git has also seemed to surpass Subversion.
+- GitHub has an enormous developer pool.
```

Comment:
i don't think that we need this section: everybody who is using github knows why scribus should be on github...